app.js

```diff
@@ -0,0 +1,96 @@
+// question 1
+
+// for a light switch statement you would use a if else statemet
+var light = 'bright'
+if(switchup === on){
+	return bright
+} else(swithDown === off){
+	return dark
+}
+
+//question 2
+// for a email you would use object 
+var user {
+	email: 'dare'
+	password: 'ginsing'
+
+}
+
+//question 3
+// you wourld make an variables
+var spaceship = ['hull', 'laserblaster', 'tractor beam', 'warp drive']
+
+/*question 4*/
+
+make an array 
+
+var students = ["jane", "Adam", "lisa", "Mark", "jesse", "allen"]
+
+
+//question 5
+
+would be a object
+
+var students = {
+class: wdi
+location: chicago
+studentName: ['yasmin', 'jeniffer', 'allen']
+}
```

Comment:
numerous syntax issues here..., if you were to use objects you'd need commas after the properties... perhaps the object would serve the purpose, 
the idea is that it would be an array of objects though